en/docs/quick-starts/adaptive-auth-overview.md

```diff
@@ -0,0 +1,20 @@
+# Adaptive Authentication Scenarios
+
+This section guides you through using pre-defined templates for common adaptive authentication usecases.
+
+   - [Role-Based](role-based-adaptive-auth.md)
+   - [User-Age-Based](age-based-adaptive-auth.md)
+   - [Tenant-Based](tenant-based-adaptive-auth.md)
+   - [User Store-Based](user-store-based-adaptive-auth.md)
+   - [IP-Based](ip-based-adaptive-auth.md)
+   - [New-Device-Based](device-based-adaptive-auth.md)
+   - [ACR-Based](acr-based-adaptive-auth.md)
+   - [Risk-Based](risk-based-adaptive-auth.md)
+   - [With Function Library](adaptive-auth-with-function-lib-sample.md)
+   - [Limiting Active User Sessions](limiting-active-sessions-adaptive-auth.md)
+   
+!!! info "Related Topics"
+    - [Concept: Adaptive-Authentication](TODO:insert-link-to-concept)
+    - [Guide: Configure Adaptive Authentication for an Application](../../guides/adaptive-auth/configure-adaptive-auth)
+    - [Guide: Adaptive Authentication Using Function Library](../../guides/adaptive-auth/adaptive-auth-with-function-lib)
+    - [Guide: Ensure Assurance with Authentication Context Class Reference (ACR)  and Authentication Method Reference (AMR)](../../guides/adaptive-auth/work-with-acr-amr)
```

Comment:
can we shorten this to "Ensure Assurance with ACR/AMR" like we've titled it in the mkdocs yaml?